### PR TITLE
Create ui-c8y-1021-0-0-allow-possibility-for-global-asset-search-when-duplicating-private-smartrule-but.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by iiar-c8y

Ticket: [MTM-54749](https://cumulocity.atlassian.net/browse/MTM-54749)
Original PR: [7324](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7324)

[MTM-54749]: https://cumulocity.atlassian.net/browse/MTM-54749?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1021-0-0-allow-possibility-for-global-asset-search-when-duplicating-private-smartrule-but.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Allow possibility for global asset search when duplicating private smartrule but update context upon duplication (#7324)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-0-0-allow-possibility-for-global-asset-search-when-duplicating-private-smartrule-but.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Allow global asset search when duplicating private smart rules and update context
```

File: content/change-logs/application-enablement/ui-c8y-1021-0-0-allow-possibility-for-global-asset-search-when-duplicating-private-smartrule-but.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-54749
+version: 1021.0.0
+---
+Allow possibility for global asset search when duplicating private smartrule but update context upon duplication (#7324)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1021-0-0-allow-possibility-for-global-asset-search-when-duplicating-private-smartrule-but.md:17]: The Claude AI suggested to use the following body:
```suggestion
Previously, when duplicating a private smart rule, the asset search was limited to the current context which made it difficult to find and select assets from other contexts. With this change, users can now search for assets globally across all contexts when duplicating a private smart rule. Upon duplication, the context of the duplicated smart rule is automatically updated to match the context of the selected assets. This improves the usability and flexibility of duplicating smart rules, allowing users to more easily create rules that span multiple contexts without manual workarounds.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by reey** [comment]
closing as already covered in: https://github.com/Cumulocity-IoT/c8y-docs/pull/2516

**Pull request closed without merging**